middleware/modelExport.js

```diff
@@ -298,7 +328,18 @@ module.exports = {
         asyncFn.push(helpers.getForm);
         asyncFn.push(helpers.getGrid);
 
-        async.waterfall(asyncFn, function (err, filters, form, grid) {
+        // get the actual export object
```

Comment:
I thought that was still a good option? Allowing them to provide an export function to replace absolutely everything...

Or do you think this multiple exports thing is good enough we shouldn't allow a complete override... I'm thinking maybe you're right...